src/mpid/ch4/src/ch4_progress.c

```diff
@@ -5,7 +5,7 @@
 
 #include "mpidimpl.h"
 
-int MPIDI_Progress_test(int flags)
+static int MPIDI_Progress_test(int flags)
```

Comment:
I am not quite sure yet that later we may still want to let other parts (e.g. posix, ofi, ) to directly call the `MPIDI_Progress_test` for specific progress. Right now I want to make it static so that I can have the flexibility and confidence to change the code here. Once settled, we may expose the function again. How about I leave a comment here and do the name space change separately later (once we understand what we want)?

Reply:
Sounds good to me.